CHANGELOG.md

```diff
@@ -36,31 +36,31 @@ All notable changes to this project will be documented in this file.
 - Improved `.lrc` lyrics files parser performance and fixed parsing issues
 - Normalized duration formatting across `QueueTimeTotal` and `QueueTimeRemaining` properties for both standard (MM:SS/H:MM:SS) and verbose formats
 - Confirm action in browsers, which either opens a directory or adds the hovered song to the queue
-no longer exhibits the latter behavior. It now instead replaces the queue with all songs in the
-directory.
+  no longer exhibits the latter behavior. It now instead replaces the queue with all songs in the
+  directory.
```

Comment:
My markdown linter is being a bit aggressive. If this is an issue I can back this out.